schema/defs.json

```diff
@@ -62,7 +62,7 @@
                     "$ref": "#/definitions/FilePath"
                 },
                 "options": {
-                    "type": "string"
+                    "$ref": "#/definitions/ArrayOfStrings"
```

Comment:
Did we miss this when we updated the spec?

Reply:
I think so. Have a vague recollection that we spotted this earlier... and I though we've merged a fix already. But apparently not.